packages/react-components/react-tree-preview/stories/Tree/TreeSelection.stories.tsx

```diff
@@ -1,20 +1,20 @@
 import * as React from 'react';
 import {
-  FlatTree as Tree,
+  FlatTree,
   TreeItem,
   TreeItemLayout,
   HeadlessFlatTreeItemProps,
   useHeadlessFlatTree_unstable,
 } from '@fluentui/react-tree-preview';
 
-const SELECTION_MODE = 'single'; // change to "multiselect" for multiselection
+const SELECTION_MODE = 'multiselect'; // change to "multiselect" for multiselection
```

Comment:
```suggestion
const SELECTION_MODE = 'multiselect';
```

The comment does not make sense